tools/cibadmin.c

```diff
@@ -120,6 +124,29 @@ static pcmk__cli_option_t long_options[] = {
         "md5-sum-versioned", no_argument, NULL, '6',
         "Calculate an on-the-wire versioned CIB digest", pcmk__option_default
     },
+#if ENABLE_ACL
+    {
+        "access-render", optional_argument, NULL, 'S',
+        "Like -Q + evaluate access for --user specified "
+            "user, presented in particular way",
```

Comment:
I'm thinking this should be named --show-access (to align with -S as the short option), and it should be used in conjunction with -Q rather than implying it -- something like "Whether to use syntax highlighting for ACLs (with -Q/--query and -U/--user)"